templates/article.liquid

```diff
@@ -128,15 +128,12 @@
           {% endif %}
 
           {% comment %}
-            Comment submission form
+            #comment-form is required, it is used as an anchor link by Shopify
+            when an error occurs during submission.
           {% endcomment %}
-          <div class="form-vertical">
+          <div id="comment-form" class="form-vertical">
```

Comment:
please remove this, as `{% form 'new_comment', article %} will dynamically generate a form with `id="comment_form"`